docs/UserGuide.md

```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
 
 ---
 
+## Structure of the User Guide
+
+Worried about how tedious it will be to read this document? Not to worry! We have structured this User Guide in a way
+that makes it easy and quick for you to find what you need. In this next subsection,
+[Reading this User Guide](#reading-this-user-guide), you can find some tips we have on how to read this guide. The
+next section, [Features](#features), documents the main functionalities of _Teaching Assistant_ and how to use them.
+
+The main functions of _Teaching Assistant_ can be summmarised as the following:
+* Contact management
+* Entry (schedules) management
+
+If you prefer a brief overview of all the commands, head over to [command summary](#command-summary) which provides you
+with a table of all available commands.
+
+### Reading this User Guide
+This section introduces you to some technical terms and syntax that will be used throughout the User Guide.
+You may want to read through this section thoroughly first before moving on to the next sections.
+
+##### Terminology
+The image of our GUI is shown below, annotated with descriptions of each GUI component we refer to in this User Guide.
+
+![Annotated GUI](images/annotatedUi.png)
+
+The table below defines some technical terminology used throughout the User Guide.
+
+Terms | Meaning
+----------------- | -------------------
+GUI | Graphical User Interface. It refers to the part of the application you interact with.
+JSON | JavaScript Object Notation. Your Teaching Assistant data is stored as a JSON file format. You can find out more on their official website [here](https://www.json.org/json-en.html)!
+
```

Comment:
Cuz I explained a bit of the terms GUI and CLI in the intro, maybe you want to shift the explanations to this table instead and remove the explanation in the intro